packages/icons-angular/.bazelrc

```diff
@@ -0,0 +1,17 @@
+# Make TypeScript compilation fast, by keeping a few copies of the compiler
+# running as daemons, and cache SourceFile AST's to reduce parse time.
+build --strategy=TypeScriptCompile=worker
+
+# Performance: avoid stat'ing input files
+build --watchfs
+
+# Do not generate the symbolic links bazel-*.
+build --symlink_prefix=dist/
+
+build --experimental_allow_incremental_repository_updates
+
+build --local_cpu_resources=1
+
+build --local_ram_resources=2000
+
```

Comment:
Can you add ?

```
build --verbose_failures
```

Also, in the circle logs, I noticed that you are running bazel build in parallel with some other build tasks or I saw wrong?

Reply:
Hi @alan-agius4,

> In CI I also suggest you to add

Will do 👍

> Also, in the circle logs, I noticed that you are running bazel build in parallel with some other build tasks or I saw wrong?

Yep, that's correct ... This repo is a monorepo for our whole design system ... We're not quite at the point where we're looking in to integrating Bazel everywhere, but it's come up before.

You can see from the build logs that even with _very_ conservative limits we're running out of memory, even after several hours worth of building.

I plan to try and reduce the number of modules we're building (by consolidating them where possible). The Angular compiler/metadata generation seems to be the bottleneck in terms of compilation time and resource consumption, so I'm not sure if simply reducing the number of modules will gain us anything.

Reply:
Yeah reducing the modules might help since you have limited CI resources.

Also I suggest you add a cache for bazel, so that after a green build bazel will only need to build the changes and not start from scratch.

See:  https://github.com/carbon-design-system/carbon/pull/3915/commits/f245a7832a5e5735f96645a662ac17fdea2c01cf

(Sry for being brief, but I am replaying from my mob)